greyhounds/edmundo.md

```diff
@@ -0,0 +1,16 @@
+---
+layout: greyhound
+title: Edmundo
+pic: edmundo.png
+sex: MALE
+age: 4 YEARS
+color: WHITE/RED
+cats: 
+permalink: /greyhounds/edmundo/
+---
+
+My name is Ed and I sometimes go by Mr. Ed, although I cannot talk. I am an inquisitive boy with the heart of a teddy bear. I love getting attention and following my foster family into every room in the house. Yes, I love houses and what a difference from my farm.
+ 
+My racing career took off once I turned of age and ended after 98 races. Now, I am retired and want to play with my people every chance I get. I was recently introduced to this thing called a doggie door. This is the best thing since ground meat, well maybe not ground meat. I can go outside anytime I want and take care of my business, bark at the deer and play with my harem. Yes, I live with three girls (smile).
```

Comment:
hahahhah good point. Ed's foster wrote that- so funny. That sentence can be
taken out.

About Horace - can he be approved with "more info coming soon?"

Thanks,
Catelyn

On Mon, Mar 16, 2015 at 7:27 AM, Zach Whaley <notifications@github.com>
wrote:

> In greyhounds/edmundo.md
> <https://github.com/gpa-centex/gpa-centex.github.io/pull/132#discussion_r26479835>
> :
>
> > @@ -0,0 +1,16 @@
> > +---
> > +layout: greyhound
> > +title: Edmundo
> > +pic: edmundo.png
> > +sex: MALE
> > +age: 4 YEARS
> > +color: WHITE/RED
> > +cats:
> > +permalink: /greyhounds/edmundo/
> > +---
> > +
> > +My name is Ed and I sometimes go by Mr. Ed, although I cannot talk. I am an inquisitive boy with the heart of a teddy bear. I love getting attention and following my foster family into every room in the house. Yes, I love houses and what a difference from my farm.
> > +
> > +My racing career took off once I turned of age and ended after 98 races. Now, I am retired and want to play with my people every chance I get. I was recently introduced to this thing called a doggie door. This is the best thing since ground meat, well maybe not ground meat. I can go outside anytime I want and take care of my business, bark at the deer and play with my harem. Yes, I live with three girls (smile).
>
> We probably shouldn't say that he is playing with a harem
> <https://www.google.com/?gws_rd=ssl#q=define+harem>
>
> I get the feeling someone could be offended.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/gpa-centex/gpa-centex.github.io/pull/132/files#r26479835>
> .
>